packages/syft/src/syft/__init__.py

```diff
@@ -159,3 +161,100 @@ def _orchestra() -> Orchestra:
 
 def search(name: str) -> SearchResults:
     return Search(_domains()).search(name=name)
+
+
+def launch(
```

Comment:
why do we need this here? isnt this duplicating code?

Reply:
We have two users for launch commands:
- very advanced PySyft developers who want all the bells and whistles, logging, config, and argument
- noobs who want to launch a server with reasonable defaults

Advanced users can keep using syft.orchestra.launch(<many arguments>).

Beginners can focus on syft.launch(<nodename>)

Reply:
The alternative is to break the syft.orchestra.launch() command to ship with all the batteries-included features beginners will want.